docs/canonicalk8s/snap/explanation/security/crypto.md

```diff
@@ -0,0 +1,96 @@
+# Cryptography
+
+This reference page provides an overview of the {{product}} cryptography
```

Comment:
Please update this with "explanation page" and provide a line or two on cryptography in canonical k8s